datahub-web-react/src/app/ingest/source/builder/RecipeForm/common.tsx

```diff
@@ -257,19 +276,44 @@ export const INCLUDE_LINEAGE: RecipeField = {
 export const INCLUDE_TABLE_LINEAGE: RecipeField = {
     name: 'include_table_lineage',
     label: 'Include Table Lineage',
-    tooltip: 'Whether or not table lineage should be ingested.',
+    tooltip: 'Extract Tabel-Level lineage metadata. Enabling this may increase the duration of the extraction process.',
     type: FieldType.BOOLEAN,
     fieldPath: 'source.config.include_table_lineage',
     rules: null,
 };
 
-export const PROFILING_ENABLED: RecipeField = {
+const isProfilingEnabled = 'source.config.profiling.enabled';
+const isTableProfilingOnly = 'source.config.profiling.profile_table_level_only';
+export const TABLE_PROFILING_ENABLED: RecipeField = {
     name: 'profiling.enabled',
-    label: 'Enable Profiling',
-    tooltip: 'Whether profiling should be performed on the assets extracted from the ingestion source.',
+    label: 'Enable Table Profiling',
+    tooltip:
+        'Generate Data Profiles for extracted Tables. Enabling this may increase the duration of the extraction process.',
+    type: FieldType.BOOLEAN,
+    fieldPath: isProfilingEnabled,
+    rules: null,
+    getValueFromRecipeOverride: (recipe: any) => {
+        return get(recipe, isProfilingEnabled);
+    },
+    setValueOnRecipeOverride: (recipe: any, value: boolean) => {
+        return setFieldValueOnRecipe(recipe, value, isProfilingEnabled);
+    },
```

Comment:
I don't think setting these override methods are necessary since `isProfilingEnabled` is set as the `fieldPath`.

Reply:
oh nice catch yes this is simple!